src/Axiom/Set/Rel.agda

```diff
@@ -80,28 +83,69 @@ mapˡ f R = map (Data.Product.map₁ f) R
 mapʳ : (B → B') → Rel A B → Rel A B'
 mapʳ f R = map (Data.Product.map₂ f) R
 
-dom∈ : ∀ {a} → a ∈ dom R ⇔ (∃[ b ] (a , b) ∈ R)
+dom∈ : ∀ {a} → (∃[ b ] (a , b) ∈ R) ⇔ a ∈ dom R
```

Comment:
Reverse order of equivalence in definition of `dom∈` to be more consistent with other definitions and so that it's more natural in the sense that `to dom∈` takes a decomposition *to* a dom inclusion.